### PR TITLE
Update Exchange-Teams-interact.md

### PR DESCRIPTION
**Pull request by RuiPereiraTabares**
updated article with information of teamsonly users presence in OWA, currently workitems for deploying this on 2021

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/Exchange-Teams-interact.md [hunk 1]
```diff
@@ -62,6 +62,7 @@ The following table provides a helpful quick reference to feature availability b
 <sup>7</sup> Teams honors the [Outlook on the web mailbox policy](https://docs.microsoft.com/powershell/module/exchange/client-access/set-owamailboxpolicy) setting that's configured by tenant admins to control whether users can change their profile picture. If the **-SetPhotoEnabled** setting is turned off in the policy, users cannot add, change, or remove their profile picture. For example, if a user uploads a profile picture that's approved by your organization's IT or HR department, no action is needed. However, if a user uploads an inappropriate picture, change it according to your organization's internal policies.
 
 <sup>8</sup> You need to meet the requirements listed in the [Requirements to create and view meetings for mailboxes hosted on-premises](#requirements-to-create-and-view-meetings-for-mailboxes-hosted-on-premises) section.
+<sup>9</sup> At the moment presence is not supported on OWA for teamsonly users.
```
Comment (JohanFreelancer9) [inline Teams/Exchange-Teams-interact.md:65]:
```suggestion

<sup>9</sup> At the moment presence is not supported on OWA for TeamsOnly users.
```

Comment (JohanFreelancer9) [reply Teams/Exchange-Teams-interact.md:65]: Please reference it in the table or insert the information as a NOTE.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@dstrome PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by dstrome** [comment]
@RuiPereiraTabares - please resubmit with the table item the footnote is referencing. Also, if you're a Microsoft employee or vendor, please link your corpnet and github accounts. thanks.

**Pull request closed without merging**